packages/core/content-releases/admin/src/components/ReleaseActionOptions.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+
+import {
+  FieldInput,
+  FieldLabel,
+  VisuallyHidden,
+  Field,
+  type FieldProps,
+} from '@strapi/design-system';
+import styled from 'styled-components';
+
+interface FieldWrapperProps extends FieldProps {
+  actionType: 'publish' | 'unpublish';
+}
+
+const getBorderLeftRadiusValue = (actionType: FieldWrapperProps['actionType']) => {
+  return actionType === 'publish' ? 1 : 0;
+};
+
+const getBorderRightRadiusValue = (actionType: FieldWrapperProps['actionType']) => {
+  return actionType === 'publish' ? 0 : 1;
+};
+
+const FieldWrapper = styled(Field)<FieldWrapperProps>`
+  border-top-left-radius: ${({ actionType, theme }) =>
+    theme.spaces[getBorderLeftRadiusValue(actionType)]};
+  border-bottom-left-radius: ${({ actionType, theme }) =>
+    theme.spaces[getBorderLeftRadiusValue(actionType)]};
+  border-top-right-radius: ${({ actionType, theme }) =>
+    theme.spaces[getBorderRightRadiusValue(actionType)]};
+  border-bottom-right-radius: ${({ actionType, theme }) =>
+    theme.spaces[getBorderRightRadiusValue(actionType)]};
+
+  > label {
+    color: inherit;
+    padding: ${({ theme }) => `${theme.spaces[2]} ${theme.spaces[3]}`};
+    text-align: center;
+    vertical-align: middle;
+    text-transform: capitalize;
+  }
+
+  &:active,
+  &.selected {
```

Comment:
using classNames doesn't feel like the "react way" to me, try data attributes maybe? That's what Radix does